src/store/index.ts

```diff
@@ -62,6 +64,12 @@ export interface IStore {
      */
     storeRoom(room: Room): void;
 
+    /**
+     * Set the user creator which is used for creating User objects
+     * @param creator - A callback that accepts an user-id and returns an User object
+     */
+    setUserCreator(creator: UserCreator): void;
```

Comment:
Temporary solution until we fix https://github.com/vector-im/element-web/issues/26383